frontend/src/app/rating/rating.page.ts

```diff
@@ -99,6 +99,30 @@ export class RatingPage implements OnDestroy, OnInit {
    */
   ngOnInit(): void {
     this.ratingForm.controls.id.disable();
+    this.storage
+      .getItem<RatingForm>(
+        this.storage.applicantRatingsDb,
+        this.ratingForm.controls.id.value
+      )
+      .then((applicantData) => {
+        if (applicantData) {
+          this.ratingForm.controls.applicantScore.controls.rhetoric.setValue(
+            applicantData.applicantScore.rhetoric
+          );
+          this.ratingForm.controls.applicantScore.controls.motivation.setValue(
+            applicantData.applicantScore.motivation
+          );
+          this.ratingForm.controls.applicantScore.controls.personalImpression.setValue(
+            applicantData.applicantScore.personalImpression
+          );
+          this.ratingForm.controls.applicantScore.controls.selfAssurance.setValue(
+            applicantData.applicantScore.selfAssurance
+          );
+          this.ratingForm.controls.impressionInfo.controls.impression.setValue(
+            applicantData.impressionInfo.impression
+          );
```

Comment:
I know you haven't requested a review from me. But one thing that I was wondering: Can't you just use `patchValue()` here? 

This would probably (careful, I haven't tested it!) look like this:

```
if (applicantData) {
  this.ratingForm.patchValue(applicantData);
}
```

Reply:
Did not know such method exist. Will change. Thanks